packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,63 +1,151 @@
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+<InfoButton content="This is some additional information." />
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+Visual (only the button has a variant)
+
+- `transparent` (default)
+
+![transparent](./etc/images/transparent-infobutton.png)
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+#### Props
+
+```ts
+export type InfoButtonSlots = {
+  root: NonNullable<Slot<'button'>>;
+
+  /**
+   * The Popover element that wraps the content and root. Use this slot to pass props to the Popover.
+   */
+  popover: NonNullable<Slot<PopoverProps>>;
+
+  /**
+   * The content to be displayed in the PopoverSurface when the button is pressed.
+   */
+  content: NonNullable<Slot<typeof PopoverSurface>>;
+};
+
+/**
+ * InfoButton Props
+ */
+export type InfoButtonProps = ComponentProps<Partial<InfoButtonSlots>>;
+
+/**
+ * State used in rendering InfoButton
+ */
+export type InfoButtonState = ComponentState<InfoButtonSlots>;
+```
 
 ## Structure
 
-- _**Public**_
-- _**Internal**_
-- _**DOM** - how the component will be rendered as HTML elements_
+_**Public**_
+
+```jsx
+<InfoButton
+  content={
+    <>
+      Popover above-start lorem ipsum dolor sit amet consectetur.
+      <Link href="https://react.fluentui.dev">Learn more</Link>
+    </>
+  }
+/>
+```
+
+_**Internal**_
+
+```jsx
+return (
+  <slots.popover {...(slotProps.popover as PopoverProps)}>
+    <PopoverTrigger>
+      <slots.root {...slotProps.root} />
+    </PopoverTrigger>
+    <slots.content {...slotProps.content} />
+  </slots.popover>
+);
+```
+
+_**DOM**_
+
+```html
+<button type="button" class="fui-Button fui-InfoButton__button">
+  <!-- icon -->
+</button>
+
+<!-- on document.body -->
+<div role="tooltip" class="fui-PopoverSurface fui-InfoButton__content">
+  Popover above-start lorem ipsum dolor sit amet consectetur.
+  <a href="https://react.fluentui.dev">Learn more</a>
+</div>
+```
 
 ## Migration
 
-_Describe what will need to be done to upgrade from the existing implementations:_
-
-- _Migration from v8_
-- _Migration from v0_
+There's no migration guide as `v0` and `v8` do not have an InfoButton component.
 
 ## Behaviors
 
-_Explain how the component will behave in use, including:_
+> Note: The behavior will change when used in conjunction with the `Field` component. It is mentioned that when a form has many infobuttons, it may be better to not make a full tab stop in the button. The idea is to have each field have a "shortcut" that will let you focus on the infobutton. This will be implemented by Field if needed.
```

Comment:
```suggestion
> Note: The behavior will change when used in conjunction with the `Field` component. It is mentioned that when a form has many InfoButtons, it may be better to not make a full tab stop in the button. The idea is to have each field have a "shortcut" that will let you focus on the infobutton. This will be implemented by Field if needed.
```